lxd/main_forkdns.go

```diff
@@ -1,75 +1,257 @@
+// forkdns provides a specialised DNS server designed for relaying A and PTR queries that cannot be
+// answered by the local dnsmasq process to the other cluster member's forkdns process where it will
+// inspect the local dnsmasq lease file looking for an answer to the query.
+// It uses the "recursion desired" flag in incoming DNS requests to modify its behaviour.
+// When "recursion desired" is set, the query is immediately relayed to the other cluster nodes
+// (with the "recursion desired" flag set to no) as it indicates that the local dnsmasq process was
+// unable to answer it from the local lease file.
+// When "recursion desired" flag is set to no, this indicates the request has been sent from another
+// forkdns process, and the local dnsmasq lease file only is parsed to try and answer the query.
```

Comment:
Wouldn't this be better to have in `cmd.Long` so it can be seen with `lxd forkdns --help`?